app/components/questions/RentBurdenRates.js

```diff
@@ -0,0 +1,216 @@
+      color: Color.Salmon
+    }
+    ]
+
+  
+  return {
+    chartData: data,
+    areaData,
+  }
+}
+
+function ExamplePieChart(props) {
+  const { data } = props
+  
+  if(!data?.[0]?.value){
+    return null;
+  }
+  
+  const tooltipStyle = {
+    backgroundColor: "white", 
+    borderColor: "LightGrey", 
+    borderStyle: "solid", 
+    borderWidth: "thin"
+  };
+  
+  
+  const CustomTooltip = ({ active, payload, label }) => {
+  if (active && payload && payload.length) {
+    // Changes the tooltip to display the formatted percentage rather than a decimal
+    return (
+      <div className="custom-tooltip" style={tooltipStyle}>
+        <p className="label">{`${payload[0].name} : ${payload[0].payload.valueFormatted}`}</p>
+      </div>
+    );
+  }
+  return null;
+  };
+  
+  
+  const RADIAN = Math.PI / 180;
+  const renderCustomizedLabel = ({ cx, cy, midAngle, innerRadius, outerRadius, color, valueFormatted, index }) => {
+    // Takes various characteristics of the pie chart to calculate the position of the label
+    const radius = 25 + innerRadius + (outerRadius - innerRadius);
+    const x = cx + radius * Math.cos(-midAngle * RADIAN);
+    const y = cy + radius * Math.sin(-midAngle * RADIAN);
+    // Returns the text label using computed x and y position, as well as provided color and formatted percentage.
+    return (
+      <text x={x}
+            y={y}
+            fill={color}
+            textAnchor={x > cx ? "start" : "end"}
+            dominantBaseline="central">
+        {`${valueFormatted}`}
+      </text>
+    );
+  };
+  
+  
+  return (
+    <ResponsiveContainer width="100%" height={400}>
+      <PieChart
+        width={400}
+        height={400}
+        margin={{ bottom: 30 }}
+      >
+        <Pie
+          data={data}
+          nameKey="label"
+          dataKey="value"
+          label={renderCustomizedLabel}
+          cx="50%"
+          cy="50%"
+          fill=""
+          outerRadius={150}
+        >
+          {data.map((entry, i) => (
+            <Cell
+              key={`cell-${i}`}
+              fill={entry.color}
+            />
+          ))}
+        </Pie>
+        <Tooltip content={<CustomTooltip />} />
+        <Legend
+          layout="horizontal"
+          verticalAlign="top"
+          align="center"
+          wrapperStyle={{ bottom: -10 }}
+        />
+      </PieChart>
+    </ResponsiveContainer>
+  );
+}
+
+function AreaSelector(props) {
+  if(!props.data){
+    return (
+      <p>Waiting for data...</p>
+    )
+  }
+  
+  const handleChange = (e) => props.handleChange(parseInt(e.target.value))
+  
+  return (
+    <div>
+      <p>Select an Area of Interest</p>
+      <select onChange={handleChange}>
+        {props.data.metrics.map((d, i) => (
+          <option key={i} value={d.area_number}>{d.name}</option>
+        ))}
+      </select>
+    </div>
+  );
+}
+
+function AreaBelongingMessage(props) {
```

Comment:
This still says belonging, do a quick sweep through the file to see if there are any other places that need to be updated to rent burdened.